haggis.js

```diff
@@ -554,3 +563,33 @@ function extractTableId(doc) {
 function sortNumeric(a, b) {
   return a - b;
 }
+
+function addPointsPerRound(game) {
+    const ptsPerRound = getTotalPointsAfterEachRound(game.rounds);
+    
+    for (let i = 0; i < game.rounds.length; i++) {
+        if (i >= game.rounds.length - 1) { 
```

Comment:
I don't think you need this if conditional if you change the loop to be `i < game.rounds.length - 1`

Reply:
good call.